frontend/elements/src/module/_groups/cards/_common/main-empty.ts

```diff
@@ -21,6 +21,17 @@ export class _ extends LitElement {
                     flex-direction: column;
                     align-items: center;
                 }
+
+                img-ui {
+                    transform: translateX(-14px);
```

Comment:
Aligns the image so that the centering feels more natural and not like its offset